packages/sn-filter-pane/src/components/FoldedListbox/FoldedListbox.tsx

```diff
@@ -20,12 +20,28 @@ export interface FoldedListboxProps {
   onClick: ({ event, resource }: FoldedListboxClickEvent) => void;
   stores: IStores;
   tabIndex?: number;
+  isInPopover?: boolean;
 }
 interface StyledGridProps {
   constraints?: stardust.Constraints;
   stardustTheme?: stardust.Theme;
   containerRef: React.RefObject<HTMLDivElement>;
 }
+interface StyledDivProps {
+  isInPopover?: boolean;
+}
+
+const StyledDiv = styled('div', { shouldForwardProp: (p) => !['isInPopover'].includes(p as string) })<StyledDivProps>(
+  ({ isInPopover }) => ({
+    '&:focus': {
+      boxShadow: 'inset 0 0 0 2px #3F8AB3 !important',
+    },
+    '&:focus-visible': {
+      outline: 'none',
+    },
```

Comment:
I have ment to asl.
do someone know why we do it like this and not ?
```
    '&:focus-visible': {
      boxShadow: 'inset 0 0 0 2px #3F8AB3 !important',
      outline: 'none',
    },
```

Reply:
https://pawelgrzybek.com/the-difference-between-css-focus-and-focus-visible-pseudo-class/#:~:text=Adding%20the%20%3Afocus%20pseudo%2Dclass,research%20in%20one%20CSS%20property.